.github/workflows/release-10_branchoff-stable.yml

```diff
@@ -92,7 +92,9 @@ jobs:
           . ./.github/scripts/release/release_lib.sh
 
           NODE_VERSION="${{ needs.prepare-tooling.outputs.node_version }}"
-          set_version "\(NODE_VERSION[^=]*= \)\".*\"" $NODE_VERSION "polkadot/node/primitives/src/lib.rs"
+          NODE_VERSION_PATTERN="\(NODE_VERSION[^=]*= \)\".*\""
+          set_version $NODE_VERSION_PATTERN $NODE_VERSION "polkadot/node/primitives/src/lib.rs"
+          set_version $NODE_VERSION_PATTERN $NODE_VERSION "cumulus/polkadot-omni-node/lib/src/nodes/mod.rs"
           commit_with_message "Bump node version to $NODE_VERSION in polkadot-cli"
```

Comment:
We should probably emit this message for both adjusted variants, or even have a commit for each?

Reply:
Agree, maybe two commits would be better here and each with its own message to keep track of it

Reply:
Done here: [c399634](https://github.com/paritytech/polkadot-sdk/pull/7367/commits/c399634881cb28074a15af261b07ce89f1a6639f)